docs/api.md

```diff
@@ -1441,6 +1463,29 @@ await page.evaluate(() => matchMedia('print').matches);
 // → false
 ```
 
+#### page.emulateNetworkConditions(networkConditions) 
+- `networkConditions` <[Object]> Network conditions to emulate.
+  - `download` <[number]> Download speed (bytes/s), `-1` to disable
+  - `upload` <[number]> Upload speed (bytes/s), `-1` to disable
+  - `latency` <[number]> Latency (ms), `0` to disable
+  - `connectionType` <?["none"| "cellular2g"| "cellular3g"| "cellular4g"| "bluetooth"| "ethernet"| "wifi"| "wimax"| "other"] Optional connection type
```

Comment:
Could you please add a bit more clarification for the params here? Are params `download`, `upload` and `latency` optional in case of having `connectionType`? Or why the `connectionType` needed at all?

Reply:
This is unrelated to the predefined network conditions in NetworkConditions.ts but only changes what `navigator.connection.type` returns. But this seems to be unsupported on desktop anyways so maybe it's best to just omit it. WDYT?

Reply:
Omitting seems reasonable for me.
Another option is to rename it to something like `connectionTypeDescription` and to explicitly describe what it intended to do.

Reply:
WDYT about the new naming?